src/SnapshotTesting.ts

```diff
@@ -0,0 +1,88 @@
+import { Input } from './Input'
+import * as lsif from './lsif'
+import { Range } from './Range'
+
+const lsiftyped = lsif.lib.codeintel.lsiftyped
+
+export function formatSnapshot(
```

Comment:
I took the liberty to extract this functionality into a separate file so that it's easier to reuse between projects.